src/api/rosetta-constants.ts

```diff
@@ -12,8 +12,11 @@ export const RosettaConstants = {
   rosettaVersion: '1.4.6',
   symbol: 'STX',
   decimals: 6,
-  lockedBalance: 'LockedBalance',
-  spendableBalance: 'SpendableBalance',
+  StakedBalance: 'StakedBalance',
+  SpendableBalance: 'SpendableBalance',
+  VestingLockedBalance: 'VestingLockedBalance',
+  VestingUnlockedBalance: 'VestingUnlockedBalance',
+  VestingSchedule: 'VestingSchedule',
```

Comment:
I see that "LockedBalance" has disapeared. Does this mean that Vesting + Staked = Locked ?

Reply:
Just to be sure there is not a 3rd way to lock tokens. The other operation that I have in mind would be multisig balance but I guess it is considered spendable...

Reply:
I've splitted the 'locked' notion of a balance into:
`StakedBalance`: locked balance due to Staking
`VestingLockedBalance`: locked balance due to vesting

So 'total locked' would be `StakedBalance` + `VestingLockedBalance`